lib/px/utils/pxconstants.lua

```diff
@@ -4,11 +4,22 @@
 -- Release date: 07.11.2016
 ----------------------------------------------
 
-local _M = {
-    MODULE_VERSION = "NGINX Module v2.12.0",
+local function readonlytable(table)
```

Comment:
this was removed on purpose in previous version. Kong plugin needs to override the `MODULE_VERSION` value, so we couldn't use the `readonlytable` for constants